srcpkgs/renderdoc/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'renderdoc'
+pkgname=renderdoc
+version=1.1
+revision=1
+build_style=cmake
+configure_args="-DENABLE_GL=ON -DENABLE_GLES=ON -DENABLE_PYRENDERDOC=ON
+-DENABLE_QRENDERDOC=ON -DENABLE=RENDERDOCCMD=ON -DENABLE_VULKAN=ON
+-DENABLE_XCB=ON -DENABLE_XLIB=ON"
+makedepends="libX11-devel libxcb-devel xcb-util-keysyms-devel libGL
+python3-devel qt5-devel qt5-svg-devel qt5-x11extras-devel bison autoconf
```

Comment:
bison, autofonf, automake and pkg-config should go into `hostmakedepends`, cross-compilation currently fails because pkg-config cannot be found.